.circleci/config.yml

```diff
@@ -20,7 +20,37 @@ jobs:
       - run:
           name: "echo openresty version"
           command: |
-            /usr/local/openresty/bin/openresty -V
+            sudo ln -s /usr/local/openresty/bin/openresty /usr/local/bin/openresty
+            openresty -V
+      - run:
+          name: "download apisix"
+          command: |
+            git clone https://github.com/apache/apisix ./apisix
+      - run:
+          name: "install apisix dependencies"
+          command: |
+            sudo apt-get install -y libldap2-dev
```

Comment:
Would be better to use the command there and wrap the environment in dockerfile?
https://github.com/api7/apisix-build-tools/blob/b5ccf73eb82aff27c2fa91477cfc91b58cc1623a/.github/workflows/package-apisix-deb-ubuntu20.04.yml#L33
https://github.com/api7/apisix-build-tools/blob/master/test/apisix/Dockerfile.test.apisix.deb.ubuntu20.04

Reply:
So, I should supoort package apisix-base and apisix first?